modules/ServerAPI/src/server/customFunctions.R

```diff
@@ -17,6 +17,73 @@ calculateTreatmemtGroupID <- function(results, inputFormat, stateGroups, resultT
   }
 }
 
+#' Move file to file server
+#'
+#' @param sourceLocations current location of files, relative from working directory or absolute
+#' @param recordedBy logged in username
+#' @param fileName name of the file to save
+#' @param entity entity object
+#' @param deleteOldFile whether to delete the old file or not
+#' @param additionalPath additional path to append to the end of the file path
+#' @return character file code or path relative from privateUploads
+#' @export
+#'
+customSourceFileMove <- function(sourceLocations, recordedBy, fileNames = NA, entityType = NA, entity = NULL, 
```

Comment:
Previously we didn't have a "customSourceFileMove" as part of the built in ACAS so we only really suported blueimp by default.  This one is essentially to match the original customSourceFileMove signature (with a few tweaks) that R code was expecting. It also makes the function handle more than one file which was not the case before. This dramatically improves image file uploads.  This function essentially just call the ACAS move file service api and sets the path to the appropriate entity folder or subfolder of the entity folder if additionalPath is passed in (image files need to go into a subfolder of the experiment folder)